app/src/main/java/com/phlox/tvwebbrowser/activity/main/MainActivity.kt

```diff
@@ -188,6 +189,14 @@ class MainActivity : AppCompatActivity() {
             homePage = it
         }
 
+        settingsModel.keepScreenOn.subscribe(this.lifecycle) {
```

Comment:
This will not compile because keepScreenOn is Boolean and not Observable. I think I will fix that problem himself.